packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -32,7 +32,7 @@ import {RecurrenceSettings} from '../TeamPrompt/Recurrence/RecurrenceSettings'
 import {ActivityDetailsRecurrenceSettings} from './ActivityDetailsRecurrenceSettings'
 
 interface Props {
-  selectedTemplateRef: ActivityDetailsSidebar_template$key | null
+  selectedTemplateRef: ActivityDetailsSidebar_template$key
```

Comment:
follow-up from your hotfix, the template is now non-nullable, which makes the logic below a little easier